spec/services/people/finder_spec.rb

```diff
@@ -16,5 +16,13 @@
         expect(people_finder.call.pluck(:id)).to eq [person.id]
       end
     end
+
+    context 'when matching nomis_offender_no filter' do
+      let(:filter_params) { { nomis_offender_no: 'ABCDEFG' } }
```

Comment:
according to the ticket, the name of this parameter should be prison_number not nomis_offender_no. We have deliberately chosen to use prison_number I believe rather than the NOMIS terminology

Reply:
@starswan you are right, sorry the ticket was not updated. I had a chat with @spikeheap and we decided to name it `nomis_offender_no` .

See https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/adr/0002-API-params-naming.md

Reply:
ok - you need to make sure that the front-end knows about the change too, otherwise we will have integration problems